### ISSUE TITLE
OpenData flow should allow multiple parallel datasets

### ISSUE THREAD

**Issue by st-randers** [Randers-Kommune-Digitalisering/vis-klimadata-initiativer-aktiviteter#95]
Due to the use of flow variables as part of the OpenData flow, it is not currently possible to run multiple flows in parallel or even consecutively. 

To sovle this issue, a queue will be implemented. The queue contains an array of datasets that should be processed separately. The next dataset in the list should be released from the queue when the previous dataset is finished processing. 

Currently, an issue with the OpenData connector prevents this ability, as no data is outputted from the connector if no new files were found. This problem is created as a separate issue which must be resolved first.

**Comment by st-randers** [comment]
Developed a queue to download and load datasets 1 at a time in OpenData flow. Status node informs how many datasets are waiting in queue.
